src/org/zalando/stups/pierone/api_v2.clj

```diff
@@ -260,8 +278,21 @@
   "get"
   [parameters request db _]
   (if-let [manifest (:manifest (first (sql/get-manifest parameters {:connection db})))]
-    (resp manifest request)
-    (resp "manifest not found" request :status 404)))
+    (let [parsed-manifest (json/decode manifest)
+          pretty (json/encode parsed-manifest {:pretty json-pretty-printer})
+          schemaVersion (get parsed-manifest "schemaVersion")
+          set-header-fn (fn [response]
+                          (cond
```

Comment:
We can use `condp` here:

~~~ clojure
(condp = schemaVersion
  1 (ring...)
  2 (ring...)
  response)
~~~